tasks/managed/collect-atlas-params/collect-atlas-params.yaml

```diff
@@ -118,6 +118,13 @@ spec:
     - name: collect-atlas-params
       image:
         quay.io/konflux-ci/release-service-utils:d320c36f3d707cd5bfe55fe783f70236c06cc2e5
+      computeResources:
```

Comment:
AFAIK, the general recommendation is that you should always set the same values for limits and requests. There was an article that Luiz shared a while back, but I can't find it. I think hares was saying the same.

Reply:
The PR I was given to guide me https://github.com/konflux-ci/build-definitions/pull/1247/files does not do that in any of its computeResources

Reply:
OK, so here's the recommendation from hares:

> * set memory requests=memory limit
> * set cpu requets
> * Do no set CPU limits

Reply:
And here's the article that Lui shared in the past: https://home.robusta.dev/blog/kubernetes-memory-limit

Reply:
This is pretty cool too - it explains why it makes sense to set cpu request, but not cpu limit: https://home.robusta.dev/blog/stop-using-cpu-limits#data-fancybox-2

Reply:
updated to remove cpu limits and set memory requests = memory limits